sonar-plugin-api/src/main/java/org/sonar/api/source/Symbolizable.java

```diff
@@ -20,14 +20,17 @@
 package org.sonar.api.source;
 
 import java.util.List;
+import org.sonar.api.batch.sensor.SensorContext;
 import org.sonar.api.component.Perspective;
 import org.sonar.api.component.ResourcePerspectives;
 
 /**
  * Use this perspective to save symbol references on files.
  * See {@link ResourcePerspectives}.
  * @since 3.6
+ * @deprecated since 5.6 use {@link SensorContext#newSymbolTable()}
  */
+@Deprecated
 public interface Symbolizable extends Perspective {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)